src/server/index.ts

```diff
@@ -1,11 +1,19 @@
 import express, { Request, Response } from 'express';
+import shareRoutes from './routes.ts/shares';
```

Comment:
the .ts on the folder name is a little odd but not the end of the world

Reply:
That definitely was an accident lol